testapps/php56_e2e/composer.json

```diff
@@ -1,6 +1,15 @@
 {
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/stanley-cheung/Protobuf-PHP"
```

Comment:
Shouldn't this get pulled in implicitly as part of the [composer requirements](https://github.com/grpc/grpc/blob/master/src/php/composer.json#L2) for grpc?